packages/experimental/Drawer/src/Drawer.types.ts

```diff
@@ -0,0 +1,200 @@
+import type { Animated, ColorValue, FlexAlignType, ModalProps, TouchableWithoutFeedbackProps, ViewProps } from 'react-native';
+
+import type { InteractionEvent, PressableFocusProps } from '@fluentui-react-native/interactive-hooks';
+
+export const DrawerName = 'Drawer';
+
+export type DrawerPositionType = 'left' | 'right' | 'bottom';
+
+export interface DrawerTokens {
+  /**
+   * The color of the backdrop.
+   * */
+  backdropColor?: ColorValue;
+
+  /**
+   * The opacity of the backdrop.
+   * */
+  backdropOpacity?: number;
+
+  /**
+   * Width of the handle.
+   */
+  handleWidth?: number;
+
+  /**
+   * Height of the handle.
+   */
+  handleHeight?: number;
+
+  /**
+   * The background color of the handle.
+   * */
+  handleBackgroundColor?: ColorValue;
+
+  /**
+   * The corner radius of the handle.
+   * */
+  handleCornerRadius?: number;
+
+  /**
+   * The top margin of the handle.
+   *  */
+  handleMarginTop?: number;
+
+  /**
+   * The bottom margin of the handle.
+   * */
+
+  handleMarginBottom?: number;
+
+  /**
+   * The start margin of the handle.
+   * */
+
+  handleMarginStart?: number;
+
+  /**
+   * The end margin of the handle.
+   * */
+
+  handleMarginEnd?: number;
+
+  /**
+   * The flex alignment of the handle.
+   * */
+  handleAlignment?: FlexAlignType;
+
+  /**
+   * The background for drawer
+   * */
+
+  drawerBackgroundColor?: ColorValue;
+
+  /**
+   * The corner radius of the Drawer
+   * @default 0
+   * */
+  drawerCornerRadius?: number;
+
+  /**
+   * The elevation of the Drawer
+   * @default 5
+   * */
+  drawerElevation?: number;
+
+  /**
+   * The position of the Drawer
+   * @default 'left'
+   * */
+  position?: 'left' | 'right' | 'bottom';
```

Comment:
Instead of having position itself a token, have left, right, bottom as individual tokens with value `DrawerTokens` and in Drawer.styling treat each token as a state condition for applying different style tokens. See Spinner size tokens for reference

Reply:
Since the value of the position token itself doesn't map to a specific style prop, you can remove this from DrawerTokens.

Reply:
Removed having it on in props.